src/testing/cluster/storage_checker.zig

```diff
@@ -164,12 +164,17 @@ pub const StorageChecker = struct {
         }
     }
 
-    pub fn replica_checkpoint(checker: *StorageChecker, superblock: *const SuperBlock) !void {
-        const syncing = superblock.working.vsr_state.sync_op_max > 0;
+    pub fn replica_checkpoint(
+        checker: *StorageChecker,
+        comptime Replica: type,
```

Comment:
Mhmmmmm I had initially kept this all within replica.zig, but was concerned that it may be too costly to do this at every checkpoint in production, especially since the number of tables in the forest increase exponentially in across LSM levels... Thoughts? Shall we only keep this on startup in replica.zig?

Reply:
Ahhh, sorry, I missed the fact that we _don't_ check this in prod at checkpoint. Yeah, I think your are correct here, so lets leave this as is!